### DISCUSSION TITLE
x-data on the element

### DISCUSSION THREAD

**Discussion by ericxin1982** [alpinejs/alpine#4139] [1. Help]
view-source:https://cenfun.github.io/turbogrid/vue-editor.html
Line76 ~ 153

Hi, 

Not sure how to ask the question, I try to get rid of vue3 stuff from this library step by step, I try to do definecomponent replacement first, like this way, how I can use alpinejs to replace it, can someone give me some hint?

Thanks
Eric Xin

**Comment by ekwoka** [comment]
You put x-data on the element in question.

**Reply by ericxin1982** [reply]
ok, did change.
